File: Commercial_Controller/Elevator.cs
using System;
using System.Collections.Generic;

namespace Commercial_Controller
{
    
    class Elevator{
        public int ID;
        public int currentFloor;
        public string direction = "IDLE";
        public List<int> requestList;
        public List<int> floorList;
        public string door = "CLOSED";
        public string BufferDirection;
        public List<int> BufferList;
        

        public Elevator (int ID, int floorCall, int requestedFloor){
            this.ID = ID;
            this.currentFloor = floorCall;
            this.requestList = new List<int>();
            this.floorList = new List<int>();
            this.BufferDirection = "UP";
            this.BufferList = new List<int>();
            for (int i = floorCall; i <= requestedFloor; i++){
                this.floorList.Add(i);
            }
        }
    }
}

File: Commercial_Controller_Java/Commercial_Controller/src/Elevator.java
import java.util.ArrayList;
import java.util.List;

/**
 * Elevator
 */
public class Elevator {
    
    public int ID;
    public int currentFloor;
    public String direction = "UP";
    public List<Integer> requestList;
    public List<Integer> floorList;
    public String door = "CLOSED";
    public String BufferDirection;
    public List<Integer> BufferList;
        

    public Elevator (int ID, int floorCall, int requestedFloor){
        this.ID = ID;
        this.currentFloor = floorCall;
        this.requestList = new ArrayList<Integer>();
        this.floorList = new ArrayList<Integer>();
        this.BufferDirection = "UP";
        this.BufferList = new ArrayList<Integer>();
        for (int i = floorCall; i <= requestedFloor; i++){
            this.floorList.add(i);
        }
    }
}